qemu/tests/cfg/live_backup.cfg

```diff
@@ -38,3 +38,12 @@
                     granularity = 32768
                 - 131072:
                     granularity = 131072
+        - data_plane:
+            only Host_RHEL.m7
+            only virtio_blk virtio_scsi
+            iothreads = iothread0
```

Comment:
@pezhang , this will add '-object iothread,id=iothread0' line, I have tried removing it, will cause error.